editor/vscode-client/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "jsonnet-lsp",
+	"description": "Code navigation for Jsonnet. Use in combination with https://github.com/liamdawson/vscode-jsonnet-language",
```

Comment:
It is the most recent incarnation but you're right that it is out of date. And since it just pulls the configuration file from the heptio repo: https://github.com/heptio/vscode-jsonnet. I'm inclined to do the same thing here and drop the requirement for another extension

Reply:
Sounds very reasonable :)